src/components/seachContainer/SearchContainer.tsx

```diff
@@ -3,19 +3,40 @@ import './searchContainer.scss';
 import { useSelector } from 'react-redux';
 import { RootStore } from '../../store';
 
-const SearchContainer = () => {
+const SearchContainer: React.FC = () => {
   const jobState = useSelector((state: RootStore) => state.job);
 
+  const listItem = [
+    'Tester',
+    'Java',
+    'PHP',
+    'Android',
+    '.NET',
+    'IOS',
+    'Business Analyst',
+    'QA QC',
+  ];
+
+  const renderList = () => {
+    return listItem.map((i) => (
+      <li className='nav-item list-inline-item'>
+        <a
+          href='#'
+          className='nav-link col-auto ms-xl-2 p-2 bg-white-hover text-white'
+        >
+          {i}
+        </a>
+      </li>
+    ));
+  };
+
   return (
-    <div
-      id='searchContainer'
-      className=' container-fluid px-5 mb-2 bg-dark text-white'
-    >
+    <div className='searchContainer container-fluid px-5 mb-2 bg-dark text-white'>
```

Comment:
Class `searchContainer` should following BEM convention (http://getbem.com/naming/)